rskj-core/src/main/java/org/ethereum/datasource/RocksDbDataSource.java

```diff
@@ -370,4 +368,14 @@ public void close() {
     public void flush() {
         // All is flushed immediately: there is no uncommittedCache to flush
     }
+
+    private static Options createOptions() {
+        Options tempOptions = new Options();
```

Comment:
just curious.. why did you call this local variable **temp** Options?

Reply:
tbh I just didn't want to use the same "options" name as in the class field. And in my mind it made sense because that variable would be temporary (being in the local scope of the method) till I return it.

Reply:
But yes, I'll change it back to "options". I guess it's more readable that way.